submissions/develabr/js-pre-oop/index.js

```diff
@@ -0,0 +1,65 @@
+const cat = {
+  species: 'cat',
+  name: 'Kitty',
+  gender: 'female',
+  legs: 4,
+  hands: 0,
+  saying: 'meow!',
+  friends: "cats don't needs any friends",
+};
+
+const catWoman = Object.create(cat);
+catWoman.species = 'human';
+catWoman.name = 'Catwoman';
+catWoman.gender = 'female';
+catWoman.legs = 2;
+catWoman.hands = 2;
+catWoman.friends = cat.name;
```

Comment:
You didn't see a problem in this task, because it's pretty simple here, but note that your `.friends` for all other objects is an array. And your `catWoman.friends` is a string for some reason. The only thing that you're doing here is printing stuff out, so it didn't invoke an error, but imagine that you did something like `.friends.map` to print every friend separately, the code wouldn't work here, since `string` doesn't have `.map()` method.

If you wish to specify that `catWoman` has only one friend, you should still keep the data type and just make it like this:
```js
catWoman.friends = [cat.name];
```